benchmark/events/ee-emit-multi-args.js

```diff
@@ -0,0 +1,20 @@
+var common = require('../common.js');
+var EventEmitter = require('events').EventEmitter;
+ 
+var bench = common.createBenchmark(main, {n: [25e4]});
+ 
+function main(conf) {
+  var n = conf.n | 0;
+ 
+  var ee = new EventEmitter();
+  var listeners = [];
+ 
+  for (var k = 0; k < 10; k += 1)
+    ee.on('dummy', function() {});
+ 
+  bench.start();
+  for (var i = 0; i < n; i += 1) {
+    ee.emit('dummy', 5, true);
```

Comment:
I don't care one way or the other. I just picked some number of arguments. I'm not sure if it'd be best to have a separate benchmark for one with tons of arguments or to lump them all into one....